packages/ui-components/src/components/search.tsx

```diff
@@ -53,16 +56,16 @@ interface IScore {
   /**
    * The indices of the text matches.
    */
-  indices: number[] | null;
+  indices?: number[];
```

Comment:
What is the reason to change this type?

Reply:
Oh oops didn't mean to leave that change in, it was leftover from a different approach. Just reverted it